RULES/cc/toolchain.go

```diff
@@ -37,14 +37,14 @@ type GccToolchain struct {
 func (gcc GccToolchain) ObjectFile(out core.OutPath, depfile core.OutPath, flags core.Flags, includes core.Paths, src core.Path) string {
 	includesStr := strings.Builder{}
 	for _, include := range includes {
-		includesStr.WriteString(fmt.Sprintf("-I%s ", include))
+		includesStr.WriteString(fmt.Sprintf("-I%q ", include))
 	}
 	for _, include := range gcc.Includes {
-		includesStr.WriteString(fmt.Sprintf("-isystem %s ", include))
+		includesStr.WriteString(fmt.Sprintf("-isystem %q ", include))
 	}
 
 	return fmt.Sprintf(
-		"%s -c -o %s -MD -MF %s %s %s %s",
+		"%q -pipe -c -o %q -MD -MF %q %s %s %q",
```

Comment:
Hah! Thanks! Does it make things faster? :)

Reply:
No idea, it's already so fast it's hard to notice any difference